bindata/v4.1.0/alerts/etcd-object-counts.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: etcd-object-counts
+  namespace: openshift-kube-apiserver
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+spec:
+  groups:
+    - name: etcd-object-counts
+      rules:
+        - alert: etcdHighObjectCounts
+          annotations:
+            summary: "etcd recorded more than 100000 objects. Latest count: {{ $value }}"
+            description: |
+              A high object count of more than 100000 objects may lead to stability issue in the API server and etcd storage stack. If this value continues to increase, it may degrade the responsiveness of the API server and etcd.
```

Comment:
This is very vague and probably not actionable.  I don't think 100k is going to be our bar.  Also, there's a huge difference between 1 object types with 100k and 100 object types with 100k

Reply:
Can you describe better actions?  How would I "fix" this alert (which is a requirement within alert guidelines https://github.com/openshift/enhancements/pull/637)